CMakeLists.txt

```diff
@@ -77,3 +79,56 @@ install(
 # install headers
 install(DIRECTORY include/pclomp
         DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
+else()
+
+# ROS2
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
+
+find_package(PCL 1.7 REQUIRED)
```

Comment:
Can we move these PCL and OpenMP  detection before ```if($ENV{ROS_VERSION} EQUAL 1)``` to avoid redundant codes?

Reply:
Thanks for the review. I have removed the redundant code.
8bf43f5